src/python/pants/backend/python/util_rules/faas.py

```diff
@@ -69,6 +69,13 @@
 logger = logging.getLogger(__name__)
 
 
+class PythonFaaSLayoutField(StringField):
+    alias = "layout"
+    valid_choices = PexVenvLayout
+    expected_type = str
+    default = PexVenvLayout.FLAT_ZIPPED.value
+
```

Comment:
It'd be good to have some help here too, e.g. something like:

```suggestion
    help = help_text(
        """
        Control the layout of the final artifact: `flat` creates a directory with the
        source and requirements at the top level, as recommended by cloud vendors,
        while `flat-zipped` (the default) wraps this up into a single zip file.
        """
    )

```